index.js

```diff
@@ -35,8 +116,18 @@ Changelog.prototype.parse = function( callback ) {
 	}.bind( this ));
 };
 
-Changelog.prototype.ticketUrl = function( id ) {
-	return this.options.ticketUrl.replace( "{id}", id );
+Changelog.prototype.ticketUrl = function( id, ticketType ) {
+	if ( !ticketType ) {
+		ticketType = this.ticketTypes[0];
+	}
+
+	return this.ticketUrlTemplates[ ticketType ].replace( "{id}", id );
+};
+
+Changelog.ticketUrlFactory = function( ticketType, thisArg ) {
```

Comment:
So you want to create the wrapping function 'manually' in the forEach loop: https://github.com/scottgonzalez/changelogplease/pull/8/files#diff-168726dbe96b3ce427e7fedce31bb0bcR188 ?